tests/fixtures/custom_task_interface_keras_regression/custom.py

```diff
@@ -81,29 +78,12 @@ def load(cls, artifact_directory):
         cls
             The deserialized object
         """
-
         with open(Path(artifact_directory) / "artifact.pkl", "rb") as fp:
             custom_task = pickle.load(fp)
 
-        custom_task.estimator.model = keras.models.load_model(Path(artifact_directory) / "model")
```

Comment:
this failed before because you saved using `model.save` (TF way of doing it), then loading with `keras.models.load_model` (old keras way of doing it), fyi